### ISSUE TITLE
Error: 'ESLint: Cannot start language service process' since last update of webstorm

### ISSUE THREAD

**Issue by wiesesascha** [eslint/eslint#9798]
* **ESLint Version: 4.14.0**
* **Node Version: v6.12.2**
* **npm Version: 3.10.10**

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint (8.1.2)

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  "extends": "airbnb",
  "parser": "babel-eslint",
  "env": {
    "browser": true,
    "es6": true,
    "mocha": true,
    "jasmine": true,
    "commonjs": true,
    "worker": true,
    "phantomjs": true,
    "jquery": true,
    "serviceworker": true
  },
  "plugins": [
    "backbone", "import", "flowtype"
  ],
  "settings": {
    "import/resolver": "webpack"
  }
```
</details>
<br>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**
I've updated my IDE **WebStorm 2017.3.2** to the newest version (Build #WS-173.4127.31) and ran into a problem concerning my automated linting with ESLint. 

I configured ESLint inside "Settings/Languages & Frameworks/JavaScript/Code Quality Tools/ESLint" with "Automatic search" for the configuration file.

In the former version the same setup worked well.

Running ESLint from terminal works as expected: 
```
./node_modules/eslint/bin/eslint.js . --ext .js --ext .jsx --cache" 
```

**What did you expect to happen?**
The IDE should highlight linting-issues instantly inside the code.

**What actually happened? Please include the actual, raw output from ESLint.**
I got an error-message 
```ESLint: Cannot start language service process```

**Comment by ilyavolodin** [comment]
That doesn't sound like any error coming from ESLint. It must be something related to WebStorm itself. There was an issue with the last release of ESLint that broke some users that were using babel-eslint. The fix was issued on babel-eslint repository and has been released about 2 weeks ago. It looks like you are running latest version, so that shouldn't be the causing issues that you are seeing, but without more specific error message, I don't think we would be able to help.

**Comment by wiesesascha** [comment]
Hey @ilyavolodin, 
thanks for your rapid reply! :+1: 

I wasn't really shure whether this issue is related to WebStorm or ESLint...

Unfortunately there's no specific error log within the message appearing on top of the files to lint.
Just the one of the whole IDE, which is quite a mass.

Should I prefer to ask WebStorm-developers in this case?

**Comment by wiesesascha** [comment]
There seems to be a problem with **flowtype** and the new release of one of WS/babel-eslint/eslint.
Because that isn't mandatory for our project, I removed this package and all exept "flow" works well. 

No real solution, but from my point of view this issue can be closed! :)

**Comment by ilyavolodin** [comment]
This has been fixed with the latest release of `babel-eslint`. It's possible that WebStorm is still using older version somehow. But yes, in general, that seems more of an issue with WebStorm setup, so I'm going to close this issue.
